roles/mysql_hardening/tasks/mysql_secure_installation.yml

```
@@ -49,14 +49,13 @@
 - name: Get all users that have no authentication_string on MySQL version >= 5.7.6 or Mariadb version >= 10.4.0
   community.mysql.mysql_query:
     query: >
-      SELECT GROUP_CONCAT(QUOTE(USER), '@', QUOTE(HOST) SEPARATOR ', ') AS users
+      SELECT CONCAT(QUOTE(USER), '@', QUOTE(HOST)) AS users
```

The query returns a list of users. In the last task we iterate over this list and do a `DROP USER {{ item.users }}`. While the query actually returns multiple users, I'd still like to change it to return a `user`-list, so the last task drops a `item.user` instead of a `item.users`. This makes it clearer for me. However I'm fine with both.

```suggestion
      SELECT CONCAT(QUOTE(USER), '@', QUOTE(HOST)) AS user
```